remote-api/src/main/java/org/alfresco/rest/api/impl/NodesImpl.java

```diff
@@ -1048,6 +1048,9 @@ else if (permissionService.hasPermission(nodeRef, perm) == AccessStatus.ALLOWED)
         node.setNodeType(nodeTypeQName.toPrefixString(namespaceService));
         node.setPath(pathInfo);
 
+
+        node.getContent().setStorageClasses(contentService.findStorageClasses(nodeRef));
```

Comment:
Please add check `if (includeParam.contains(PARAM_INCLUDE_STORAGE_CLASSES))` before requesting the storage classes info as per specs.... This action is performance intense and should not be done without request.

Reply:
Done.